src/app/util/ember-compatibility-functions.cpp

```diff
@@ -234,12 +234,14 @@ CHIP_ERROR ReadSingleClusterData(FabricIndex aAccessingFabricIndex, const Concre
     AttributeAccessInterface * attrOverride = findAttributeAccessOverride(aPath.mEndpointId, aPath.mClusterId);
     if (attrOverride != nullptr)
     {
+        ConcreteReadAttributePath readPath(aPath);
```

Comment:
This is OK for now, I guess but we should push this up to the consumer....  Followup.

Reply:
https://github.com/project-chip/connectedhomeip/issues/12049